pkgs/development/python-modules/qiskit-terra/default.nix

```diff
@@ -0,0 +1,69 @@
+{ lib
+, isPy3k
+, buildPythonPackage
+, fetchPypi
+, jsonschema
+, marshmallow
+, marshmallow-polyfield
+, networkx
+, numpy
+, pillow
+, ply
+, psutil
+, requests
+, requests_ntlm
+, scipy
+, sympy
+, IBMQuantumExperience
+, vcrpy
+, jupyter
+}:
+
+buildPythonPackage rec {
+  pname = "qiskit-terra";
+  version = "0.7.0";
+
+  disabled = !isPy3k;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "9cc57da08896627d0f34cf6ae76bd3358a5e6e155f612137ff343dd787134720";
+  };
+
+  patches = [
+    ./marshmallow-polyfield.patch
```

Comment:
Please make an upstream PR with this patch and add a comment with the link.

Reply:
I adopted an appropriate version of `marshmallow_polyfield` so this patch is not needed anymore.

Reply:
You should still open an upstream PR because our next automatic upgrade will break qiskit-terra otherwise.